app/assets/stylesheets/tinymce/skins/alchemy/skin.min.css.scss

```diff
@@ -21,12 +22,13 @@
   line-height: normal;
   font-weight: normal;
   text-align: left;
-  @include box-sizing(content-box);
+  box-sizing: content-box;
   direction: ltr;
 }
 
 .mce-widget button {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
+  @include button($base-color: $white, $padding: none);
```

Comment:
Please remove the `button` mix in from here. The actual styles are set on `.mce-btn` (see the comment below, because it wasn't working right now)